install_config/certificate_customization.adoc

```diff
@@ -284,6 +284,64 @@ openshift_hosted_registry_routetermination=reencrypt <3>
 +
 * Set to `passthrough` to terminate encryption at the destination. The destination is responsible for decrypting traffic.
 
+[[configuring-custom-certificates-lb]]
+== Configuring a Custom Certificate for a Load Balancer
+
+If your {product-title} cluster uses the default load balancer or an enterprise-level load balancer, 
+you can use custom certificates to make the web console and API available externally using a publicly-signed custom certificate, leaving the existing internal certificates for 
+the internal endpoints.
+
+To configure {product-title} to use custom certificates in this way:
+
+. Edit the `servingInfo` section of the xref:../install_config/master_node_configuration.adoc#master-configuration-files[master configuration file]:
++
+----
+servingInfo:
+  logoutURL: ""
+  masterPublicURL: https://openshift.example.com:8443
+  publicURL: https://openshift.example.com:8443/console/
+  namedCertificates:
+    bindAddress: 0.0.0.0:8443
+    bindNetwork: tcp4
+    certFile: master.server.crt
+    clientCA: ""
+    keyFile: master.server.key
+    maxRequestsInFlight: 0
+    requestTimeoutSeconds: 0
+    namedCertificates:
+      - certFile: wildcard.example.com.crt <1>
+        keyFile: wildcard.example.com.key <2>
+        names:
+          - "openshift.example.com"
+  metricsPublicURL: "https://metrics.os.example.com/hawkular/metrics"
+----
++
+<1> Path to the certificate file for the web console.
++
+<2> Path to the key file for the web console.
++
+[NOTE]
+====
+The `namedCertificates` section should be configured for the host name associated with the `masterPublicURL` and `oauthConfig.assetPublicURL` settings only. 
```

Comment:
Configure the  `namedCertificates` section for only the host name that is associated with the `masterPublicURL` and `oauthConfig.assetPublicURL` settings.